eng/mark-shipped.ps1

```diff
@@ -0,0 +1,51 @@
+[CmdletBinding(PositionalBinding=$false)]
+param ()
+
+Set-StrictMode -version 2.0
+$ErrorActionPreference = "Stop"
+
+function MarkShipped([string]$dir) {
+    $shippedFilePath = Join-Path $dir "PublicAPI.Shipped.txt"
+    $shipped = Get-Content $shippedFilePath
```

Comment:
IDK which powershell you are targetting with this script, but if not 7+ I would set the encoding explicitly.